pkg/kubelet/dockertools/manager.go

```diff
@@ -64,6 +64,8 @@ const (
 	// we want to able to consider SRV lookup names like _dns._udp.kube-dns.default.svc to be considered relative.
 	// hence, setting ndots to be 5.
 	ndotsDNSOption = "options ndots:5\n"
+
+	containerNetworkEnvKey = "NETWORK"
```

Comment:
First, env vars in general are a really bad way of sharing information.  Second, we can not just assume control of an env var - the general pattern for these things is that we offer an API by which a user can insert our data into their own env var explicitly.